communities

Lukong123 opened the pull request:
Still working on it, please I need reviews to make it better.

rossbar reviewed (commented):
Thanks for this @Lukong123 , I went ahead and took the liberty of pushing up a few changes to get the development/site integration part unstuck:
 - In 695213a I've added the notebook to the site toctree, so it will automatically be executed and included in the website now.
 - In 97c4460 I converted the notebook into `myst` format using `jupytext --from markdown --to myst`. Note that this is also necessary in order for the notebook to be executed during the website build. No further changes are needed on this front :)
 - I removed the two saved static images - now that the notebook is being executed, these images are being generated automatically during execution, so there's no need to store a saved copy of them in the repo.

Note that now that the notebook is running, there are some notebook failures - particularly with the `lukes_partitioning` method. I'll leave it to you to figure out what's going on there!

Lukong123 commented:
> Thanks for this @Lukong123 , I went ahead and took the liberty of pushing up a few changes to get the development/site integration part unstuck:
> 
> * In [695213a](https://github.com/networkx/nx-guides/commit/695213ab4b7beeff782f4aaa3b1e18d4aecb51fe) I've added the notebook to the site toctree, so it will automatically be executed and included in the website now.
> * In [97c4460](https://github.com/networkx/nx-guides/commit/97c4460c3007cb5b984e39dd24af424ffd53246a) I converted the notebook into `myst` format using `jupytext --from markdown --to myst`. Note that this is also necessary in order for the notebook to be executed during the website build. No further changes are needed on this front :)
> * I removed the two saved static images - now that the notebook is being executed, these images are being generated automatically during execution, so there's no need to store a saved copy of them in the repo.
> 
> Note that now that the notebook is running, there are some notebook failures - particularly with the `lukes_partitioning` method. I'll leave it to you to figure out what's going on there!

Thanks sir, for the correction, I did not consider the fact that `lukes_partitioning` works on `trees` I will adjust some of the codes as some of them are incomplete. Thank you sir.

rossbar commented:
This one has been stale for a while so I'm going to go ahead and close it. @Lukong123 if you are still interested in contributing this notebook, the place to start would be by addressing the previous comments from review, then get the notebook build passing locally - please feel free to reopen after that!
